src/codeu/chat/client/ClientUser.java

```diff
@@ -44,19 +45,21 @@ public ClientUser(Controller controller, View view) {
     this.view = view;
   }
 
-  // Validate the username string
-  static public boolean isValidName(String userName) {
-    boolean clean = true;
-    if (userName.length() == 0) {
-      clean = false;
-    } else {
 
-      // TODO: check for invalid characters
+  // Makes sure the username/password combo doesn't contain any bad chars
+  static public boolean isValidUserInput(String username, String password) {
```

Comment:
nit: I'd put the public keyword before the static keyword, or at least be consistent with other members of this class. (You used public static above.)

I'd also add some documentation on what it means for a username and a password to be valid. Is this checking that they match? What characters are you disallowing? Do usernames and passwords have the same character restrictions? Why are these being checked together instead of separately?

If `isValidInput()` is public, and this function pretty much just calls that, why do we need this function?